manifests/nodejs.yml

```diff
@@ -860,7 +860,7 @@ tests/:
         nextjs: missing_feature
       Test_Blocking_request_body_multipart:
         '*': *ref_5_25_0
-        fastify: missing_feature
+        fastify: *ref_5_57_0
```

Comment:
careful to double check we merged the feature in this version before merging this PR